.devops/cstar-rtd-deploy-pipelines.yml

```diff
@@ -49,7 +49,7 @@ variables:
     containerRegistry: '$(UAT_CONTAINER_REGISTRY_NAME)'
     selfHostedAgentPool: $(UAT_AGENT_POOL)
 
-  ${{ elseif or(startsWith(variables['Build.SourceBranch'], 'refs/heads/main'), eq(parameters.ENV, 'PROD')) }}:
+  ${{ elseif startsWith(variables['Build.SourceBranch'], 'refs/heads/main') }}:
```

Comment:
@lucaconsalvi the parameter ENV is missing in this pipeline, is it necessary to include this condition?

Reply:
I checked the others services and we don't have it in all off them, but in some of these we have this or() that I supposed that it was an old pieces of code. So I decided to change this line in all the services that have it. Let me know if it sounds good for you or we'll take care of this point.